.github/workflows/build.yml

```diff
@@ -16,6 +16,9 @@ jobs:
   build:
     name: build
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
```

Comment:
Is this permission required?

Reply:
Yep, the attestation workflow require `id-token` to be specified: https://github.com/actions/attest-build-provenance#:~:text=The%20id%2Dtoken%20permission%20gives%20the%20action%20the%20ability%20to%20mint%20the%20OIDC%20token%20necessary%20to%20request%20a%20Sigstore%20signing%20certificate.%20The%20attestations%20permission%20is%20necessary%20to%20persist%20the%20attestation.